content/learn/05.communication/01.wire/wire.md

```diff
@@ -1,30 +1,92 @@
+
+### What's the difference between the two?
+Both Qwiic and STEMMA QT use I2C, and even when inspecting modules using the two standards up close, it may be difficult to tell what makes them unique from each other. But there is a difference! And it has some implications on how and for what you may use them.
+
+Qwiic has level shifting and voltage regulation on the controller (but not on the peripherals). What this means is that Qwiic is 3.3 V logic **only**. This makes it easier to use, as for the end user, there is one less thing that can go wrong when designing and assembling your circuit.
+
+STEMMA QT, on the other hand, doesn't have this. This lets you use both 3.3 V and 5 V logic for modules. This also means that there is one more thing you may need to consider when creating your circuit, but it also grants some more flexibility in power and logic requirements.
+
+The pin order for STEMMA QT is designed to match the pin order for Qwiic, enabling cross-compatibility between the two standards
+
+## Wire Library
+The Wire library is what Arduino uses to communicate with I2C devices. It is included in all board packages, so you don't need to install it manually in order to use it.
+
+To see the full API for the Wire library, visit its [documentation page](https://www.arduino.cc/reference/en/language/functions/communication/wire/).
+
+- `begin()` - Initialise the I2C bus
+- `end()` - Close the I2C bus
+- `requestFrom()`- Request bytes from a peripheral device
+- `beginTransmission()` - Begins queueing up a transmission 
+- `endTransmission()` - Transmit the bytes that have been queued and end the transmission 
+- `write()` - Writes data from peripheral to controller or vice versa
+- `available()` - returns the number of bytes available for retrieval
+- `read()` - Reads a byte that was transmitted from a peripheral to a controller.
+- `setClock()` - Modify the clock frequency
+- `onReceive()` - Register a function to be called when a peripheral receives a transmission
+- `onRequest()` - Register a function to be called hwen a controller requests data
+- `setWireTimeout()` - Sets the timeout for transmissions in controller mode
+- `clearWireTimeoutFlag()` - Clears the timeout flag
+- `getWireTimeoutFlag()` - Checks whether a timeout has occurred since the last time the flag was cleared.
+
+## Examples
+The remainder of this article is a collection of links to tutorials, schematics and code snippets that can get you off the ground with I2C.
 
 ***Please note that the I2C bus is attached to different pins depending on the board you are using. For example, the pins used for [MKR WiFi 1010](/hardware/mkr-wifi-1010) are D11, D12, while the pins for [UNO](/hardware/uno-rev3) are D18, D19. See the image below to understand how to locate the correct pins on your board.***
 
 ![I2C pins on the UNO.](assets/uno-i2c.png)
 
 ![I2C pins on the MKR WiFi 1010](assets/wifi-1010-i2c.png)
 
-## Tutorials
-
+## Tutorials 
```

Comment:
We should spend some time updating this section, maybe we can combine it with what I proposed in the "Stemma/Qwiic" section, and provide some newer examples for popular breakouts. This is outside the original scope so let's talk more about this next week.